.github/workflows/build-lint-test.yml

```
@@ -0,0 +1,137 @@
+name: Build, Lint and Test
```

Absolute nitpick: Please always use the Oxford comma.
```suggestion
name: Build, Lint, and Test
```
Or `Build, Lint, Test`